integrationtest-service-framework/src/main/java/org/hypertrace/core/serviceframework/config/IntegrationTestConfigClient.java

```diff
@@ -36,13 +36,22 @@ public Config getConfig() {
 
   @Override
   public Config getConfig(String service, String cluster, String pod, String container) {
-    return loadConfig(service, defaultTestName)
-        .withFallback(loadConfig(service, cluster, pod, container))
-        .withFallback(loadConfig(service, cluster, pod))
-        .withFallback(loadConfig(service, cluster))
-        .withFallback(loadConfig(service))
-        .withFallback(loadConfig(INTEGRATION_TEST_COMMON_DIRECTORY))
-        .resolve();
+    if (defaultTestName.isPresent()) {
```

Comment:
nit - can do 
```java
return defaultTestName.map(testName -> loadConfig...)
  .orElse(ConfigFactory.empty())
  .withFallback(loadConfig(service, cluster, pod, container))
  ...
  ```
Avoids the explicit `get()` and repetition.

Reply:
Done